src/pages/FRAnime/main.ts

```diff
@@ -0,0 +1,93 @@
+import { pageInterface } from '../pageInterface';
+
+export const FRAnime: pageInterface = {
+  name: 'FRAnime',
+  domain: 'https://franime.fr',
+  languages: ['French'],
+  type: 'anime',
+  isSyncPage(url) {
+    return $('div#episode').length === 0;
+  },
+  isOverviewPage(url) {
+    return Boolean($('div#episode').length);
+  },
+  sync: {
+    getTitle(url) {
+      const videoSchema = JSON.parse(
+        $('div > script[type="application/ld+json"]:contains("VideoObject")').first().text(),
```

Comment:
If video is reloaded sometimes not there yet and results in a error. Has to be checked in the init